src/lib.rs

```diff
@@ -17,11 +17,11 @@ pub mod utransport;
 use protobuf::{Enum, Message};
 use std::{
     collections::HashMap,
-    sync::{atomic::AtomicU64, Arc, Mutex},
+    sync::{Arc, Mutex},
```

Comment:
When I've been developing async code I've starting using the `Mutex` implementation provided by the async framework because then the `lock()` is async aware so it's possible to hold across an `.await` point.

Maybe use `async_std::sync::Mutex`? What are your thoughts?

Reply:
If we want to hold the `Mutex` across a `.await` point, we should use `async_std::sync::Mutex`.
But in my case, I don't need to hold the Mutex. I just want to make sure the mutual exclusivity of `HashMap` and no matter insert / remove can be done without holding the Mutex across `.await` point.

I can still use `async_std::sync::Mutex` in this case, but then I need to make sure the Mutex is always used in the async function, which increase the complexity.

Reply:
Ah okay. Interesting trade-off. Thanks for the reasoning. :+1: